app/controllers/passwords_controller.rb

```diff
@@ -0,0 +1,41 @@
+class PasswordsController < ApplicationController
+
+  def new
+  end
+
+  def create
+    email = params[:email]
+    user = User.find_by_email(email)
```

Comment:
The dynamic finders are no longer recommended, it is recommended that you use `find_by` instead:

```ruby
user = User.find_by(email: email)
```